pkg/services/object_manager/storagegroup/collect.go

```diff
@@ -58,5 +58,5 @@ func CollectMembers(r objutil.HeadReceiver, cnr *cid.ID, members []oidSDK.ID) (*
 	sg.SetValidationDataSize(sumPhySize)
 	sg.SetValidationDataHash(cs)
 
-	return sg, nil
+	return &sg, nil
```

Comment:
also, i would change the return type here too. otherwise, we risk forgetting to change that later

Reply:
https://github.com/nspcc-dev/neofs-node/pull/1389#discussion_r875667403